python/xorbits/pandas/mars_adapters/__init__.py

```diff
@@ -101,3 +102,20 @@ def _install():
         MarsDataFrameGroupBy.transform, "transform", DataType.dataframe_groupby
     )
     register_data_members(DataType.dataframe_groupby, dataframe_groupby_udfs)
+
+    # install iteration functions
+    # DataFrame.itertuples, DataFrame.itertuples
+    dataframe_iteration_functions = dict()
+    dataframe_iteration_functions["itertuples"] = wrap_iteration_functions(
+        MarsDataFrame.itertuples, "itertuples", True
+    )
+    dataframe_iteration_functions["iterrows"] = wrap_iteration_functions(
```

Comment:
Do we also need to wrap DataFrame.items?

https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.items.html#pandas.DataFrame.items

Reply:
`DataFrame.items` Iterates over the DataFrame columns, it won't trigger execution in Mars, I prefer to keep this behavior as one column data would be huge, return a Xorbits Series is better.